frontend/public/components/storage/create-pvc.tsx

```diff
@@ -0,0 +1,331 @@
+/* eslint-disable no-undef */
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+
+import { k8sCreate, K8sResourceKind, referenceFor } from '../../module/k8s';
+import { ButtonBar, history, LoadingBox, NameValueEditor, resourceObjPath, RequestSizeInput } from '../utils';
+import { ListDropdown } from '../RBAC/bindings';
+import { RadioInput } from '../radio';
+import { Checkbox } from '../checkbox';
+import { PersistentVolumeClaimModel } from '../../models/index';
+
+const StorageClassDropdown: React.SFC<StorageClassDropdownProps> = props => {
+  const kind = 'StorageClass';
+  const resources = [{ kind }];
+  const { selectedKey, required, namespace } = props;
+  return (
+    <ListDropdown
+      {...props}
+      desc="Storage Classes"
+      resources={resources}
+      selectedKeyKind={kind}
+      placeholder="Select storage class"
+      selectedKey={selectedKey}
+      required={required}
+      namespace={namespace}
+      name={name}
+    />
+  );
+};
+// This form is done a little odd since it is used in both its own page and as
+// a sub form inside the attach storage page.
+export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVCFormState> {
+  state = {
+    storageClass: '',
+    pvcName: '',
+    accessMode: 'ReadWriteOnce',
+    requestSizeValue: '1',
```

Comment:
Should I remove the placeholder as well then?  The placeholder right now appears as '1', not the value so it looks filled in, but because I have the default it works.

Reply:
Yeah, I'd remove the placeholder here, too.